app/assets/v2/css/profile.css

```diff
@@ -306,3 +310,84 @@
 .no-profile-textcolor {
   color: #666666;
 }
+
+#float_avatar_image{
+  z-index: 9999999;
+  margin-top: -140px;
+}  
+
+#float_avatar_image img{
+  height: 186px;
+  width: 186px;
+  border: 3px solid #FFFFFF;
+  box-shadow: 0 -2px 4px 0 rgba(0,0,0,0.50);
+}  
+
+#float_avatar_image.is_my_profile:hover img{
+  border: 3px solid #0fce7c;
+}  
+
+.is_my_profile{
+  cursor: pointer;
+}
+#avatar_edit{
+  text-align: center;
+  background-color: white;
+  border-radius: 50%;
+  opacity: 0.4;
+  filter: alpha(opacity=40);
+  color: black;
+  vertical-align: text-top;
+  height: 186px;
+  width: 186px;
+}
+#avatar_edit_text {
+  position: absolute;
+  top: -25px;
+  left: 55px;
+  color: black;
+  font-weight: bold;
+  z-index: 999999;
+  font-size: 20px;
+}
+#avatar_edit:hover {
+  opacity: 0.7;
+  filter: alpha(opacity=70);
+}
+#avatar_edit:hover #avatar_edit_text {
+  color: #aaaaaa;
+}
+
+@media (max-width: 1500px) {
+    #float_avatar_image{
+    }  
+}
+
+
+@media (max-width: 800px) {
+    #float_avatar_image{
+      margin-top: -270px;
+    }  
+}
+
+@media (max-width: 790px) {
+    #float_avatar_image{
+      margin-top: -260px;
+    }  
+    #avatar_edit {
```

Comment:
using bootstrap display clases will be better I think. like d-md-none will hide the element when md media query is displayed.
Also I think now is ok but using media queries is always better to go mobile first, so instead of max-width 790px going min-width 790px and the default value of margin-top in float_avatar_image with the one needed to the smallest display

Reply:
fixed